core/cmd/local_client.go

```diff
@@ -143,6 +143,7 @@ func (cli *Client) runNode(c *clipkg.Context) error {
 	var fileErr error
 	vrfPasswordFile := c.String("vrfpassword")
 	if len(vrfPasswordFile) != 0 {
+		// TODO: deprecate when config V2 is rolled out.
```

Comment:
I think it may be desirable to continue to support this

Reply:
OK in that case I will treat is as override.